### ISSUE TITLE
Does the model use data augmentation strategies when training?

### ISSUE THREAD

**Issue by ahong007007** [xingyizhou/UniDet#10]
Thank you for your hard work and spiritual welfare. Does the model use data augmentation strategies when training? How to increase the data augmentation strategy?

Thank you very much.

**Comment by xingyizhou** [comment]
Yes, we use the default detectron2 data augmentation (random resize) specified [here](https://github.com/xingyizhou/UniDet/blob/master/projects/UniDet/configs/Base-CRCNN-COCO.yaml#L46). 
You can increase the sizes here (usually increase from 480-960 helps). You can also enable random crops. See the detectron2 documentations for detauls.
